Sources/SwiftDiagnostics/DiagnosticsFormatter.swift

```diff
@@ -139,10 +179,10 @@ public struct DiagnosticsFormatter {
     switch message.severity {
     case .error:
       let annotation = ANSIAnnotation(color: .red, trait: .bold)
-      return annotation.applied(to: message.message)
+      return annotation.applied(to: "error: \(message.message)")
     case .warning:
       let annotation = ANSIAnnotation(color: .yellow)
-      return annotation.applied(to: message.message)
+      return annotation.applied(to: "warning: \(message.message)")
```

Comment:
Thank you! Should we bold the `error` and `warning` bits to make them stand out more?

Reply:
Good idea. The `error` text should already be bold (as part of the message that gets annotated), I'll add the necessary changes for warnings as well 👍🏻